core/ir/aarch64/instr_create_api.h

```diff
@@ -129,6 +129,9 @@
 /** Create an operand specifying MUL, a multiplier operand. */
 #define OPND_CREATE_MUL() opnd_add_flags(OPND_CREATE_INT(DR_SHIFT_MUL), DR_OPND_IS_SHIFT)
 
+/** Create an operand specifying a condition code. */
+#define OPND_CREATE_COND(code) opnd_add_flags(OPND_CREATE_INT(code), DR_OPND_IS_CONDITION)
```

Comment:
This should explain what valud this "code" should take.  I think it should take in a #dr_pred_type enum and subtract one: that would be much easier to use, right?

Reply:
I have replaced the macro with `opnd_t opnd_create_cond(dr_pred_type_t)` and `dr_pred_type_t opnd_get_cond(opnd_t)` functions so we can convert between `dr_pred_type_t` and condition code operands and provide more helpful error messages if a user tries to create a cond operand with an invalid `dr_pred_type_t` value.